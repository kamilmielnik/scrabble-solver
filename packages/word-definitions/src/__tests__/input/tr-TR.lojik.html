[{"madde_id":"44658","kac":"0","kelime_no":"33889","cesit":"0","anlam_gor":"0","on_taki":null,"on_taki_html":null,"madde":"lojik","madde_html":"<strong>lojik<\/strong>","cesit_say":"0","anlam_say":"3","taki":"ği","cogul_mu":"0","ozel_mi":"0","egik_mi":"0","lisan_kodu":"13","lisan":"Fransızca logique","telaffuz_html":null,"lisan_html":null,"telaffuz":"l ince okunur","birlesikler":null,"font":null,"madde_duz":"lojik","gosterim_tarihi":null,"anlamlarListe":[{"anlam_id":"58622","madde_id":"44658","anlam_sira":"1","fiil":"0","tipkes":"0","anlam":"Mantıkla ilgili","anlam_html":null,"gos":"0","gos_kelime":"0","gos_kultur":"0","ozelliklerListe":[{"ozellik_id":"20","tur":"3","tam_adi":"sıfat","kisa_adi":"sf.","ekno":"31"}]},{"anlam_id":"58623","madde_id":"44658","anlam_sira":"2","fiil":"0","tipkes":"0","anlam":"► mantıklı","anlam_html":"<p>► mantıklı<\/p>\n","gos":"0","gos_kelime":"0","gos_kultur":"0","orneklerListe":[{"ornek_id":"54500","anlam_id":"58623","ornek_sira":"1","ornek":"Osman mekanik araçları, bu araçların işleyişini göz önüne alarak aslına uymayı gözden kaçırmayan lojik bir sorumlulukla tasvir etmiştir.","kac":"1","yazar_id":"2221","yazar_vd":"","yazar":[{"yazar_id":"2221","tam_adi":"Sezer Tansuğ","kisa_adi":"S. Tansuğ","ekno":"2454"}]}]},{"anlam_id":"58624","madde_id":"44658","anlam_sira":"3","fiil":"0","tipkes":"0","anlam":"► mantık","anlam_html":"<p>► mantık<\/p>\n","gos":"1","gos_kelime":"0","gos_kultur":"0","ozelliklerListe":[{"ozellik_id":"19","tur":"3","tam_adi":"isim","kisa_adi":"a.","ekno":"30"},{"ozellik_id":"57","tur":"1","tam_adi":"felsefe","kisa_adi":"fel.","ekno":"106"}]}]}]
