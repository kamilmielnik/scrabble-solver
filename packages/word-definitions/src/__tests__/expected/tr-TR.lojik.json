{
  "definitions": ["Mant覺kla ilgili", "mant覺kl覺", "mant覺k"],
  "exists": true
}
